Resources/Prototypes/_CD/Reagents/medicine.yml

```diff
@@ -707,4 +713,139 @@
+          - "reagent-effect-anxietymed-normaloverdose6"
+        probability: 0.06
+        conditions:
+        - !type:ReagentThreshold
+          min: 10.1
+      - !type:ResetParacusia
+        conditions:
+        - !type:ReagentThreshold
+          min: 10
+      - !type:MovespeedModifier
+        probability: 0.3
+        conditions:
+        - !type:ReagentThreshold
+          min: 10.1 # Slight overdose slows you down a bit
+        walkSpeedModifier: 0.85
+        sprintSpeedModifier: 0.75
+      - !type:GenericStatusEffect
+        probability: 0.1
+        conditions:
+        - !type:ReagentThreshold
+          min: 20 # taking far too much has a chance to knock you out
+        key: ForcedSleep
+        component: ForcedSleeping
+        refresh: false
+        type: Add
+      - !type:PopupMessage
+        type: Local
+        visualType: Medium
+        messages:
+          - "reagent-effect-antipsych-fade" # A decent chance to notify the player when their meds run out
+        probability: 0.5
+        conditions:
+          - !type:ReagentThreshold
+            max: 0.1
+
+- type: reagent
+  id: Rememberium
+  name: reagent-name-rememberium
+  group: Medicine
+  desc: reagent-desc-rememberium
+  physicalDesc: reagent-physical-desc-fizzy-and-creamy
+  flavor: medicine
+  color: "#FAE0BA"
+  metabolisms:
+    Medicine:
+      metabolismRate : 0.01
+      effects:
+      - !type:PopupMessage
+        type: Local
+        visualType: Large
+        messages:
+          - "reagent-effect-anti-amnesia-normal1"
+          - "reagent-effect-anti-amnesia-normal2"
+          - "reagent-effect-anti-amnesia-normal3"
+          - "reagent-effect-anti-amnesia-normal4"
+        probability: 0.085
+      - !type:HealthChange # poisons you on OD
+        conditions:
+        - !type:ReagentThreshold
+          min: 6
+        damage:
+          types:
+            Poison: 1.0
+      - !type:Paralyze
+        conditions:
+        - !type:ReagentThreshold
+          reagent: Saprodipine # wildly incompatible with antipsychotic medications.
+          min: 5
+      - !type:AdjustReagent
+        reagent: Saprodipine # purges Saprodipine so paralysis isn't too long
+        amount: -0.2
+      - !type:Drunk # OD causes drunkeness
+        conditions:
+          - !type:ReagentThreshold
+            min: 5
+      - !type:AdjustReagent
+        conditions:
+        - !type:ReagentThreshold
+          reagent: Ethanol # Alcohol makes it really easy to overdose by increasing your body's absorption of the medicine.
+          min: 1
+        reagent: Rememberium
+        amount: 0.02
+      - !type:AdjustReagent
+        probablity: 0.8 # Chance of Addiction rising per tick
+        conditions:
+        - !type:ReagentThreshold
+          reagent: Addictine
+          max: 10 # maximum addiction severity for this reagent. A moderate amount of addictiveness.
+        - !type:ReagentThreshold
+          min: 4
```

Comment:
this will cause addiction to happen only if the metabolizer is already addicted. unsure if intentional.

Reply:
nevermind to both of these im a moron